unit_testing/test_labels.py

```diff
@@ -216,3 +233,17 @@ def test_remove_other_labels_from_image(test_image):
     diff = res.data == expected.data
 
     assert diff.all()
+
+
+@pytest.mark.parametrize("test_image", test_images)
+def test_find_missing_label(test_image):
+    img = test_image.copy()
```

Comment:
Ditto about consistency for this. In one test, `test_images` is used, and for the other test, only `fake_3dimage_sct` is used.

Reply:
You are right, I think `test_image` might more complete (and I used a few tests in this file)  but the MSE is only tested on `fake_3d_image_sct` (like other tests in this file). Do you have an opinion on which one should be used? Should I correct the rest of the test as well for the sake of consistency?

Reply:
I think in general it's good to parametrize only when the differences in inputs are relevant to the code being tested.  

In this case, the only thing the `find_missing_label` does is compare lists of coordinate values, so I don't think the difference between 3d/2d is very relevant. Using only 1 image should be sufficient, I think.

Reply:
Right, I used `fake_3d_image_sct`